cogs/tags.py

```diff
@@ -219,23 +219,23 @@ async def tag(self, ctx, *, tag_name: TagViewConverter = None):
 			return
 
 		tag_name, record = tag_name
-		await ctx.send(record.get('content'))
+		await ctx.send(record.get('content'), allowed_mentions=discord.AllowedMentions.none())
 
 		await self.db.execute(
 			'UPDATE tag SET uses=$2, viewed_at=$3 WHERE id=$1',
 			record.get('id'), record.get('uses') + 1, datetime.utcnow()
 		)
 
 	@tag.command(aliases=['add', 'new'])
-	async def create(self, ctx, tag_name: tag_create_converter, *, content: commands.clean_content = None):
+	async def create(self, ctx, tag_name: tag_create_converter, *, content: str = None):
```

Comment:
Why isn't content being cleaned anymore?

Reply:
Content no longer needs to be cleaned as by utilizing allowed mentions it won't actually ping any mentions in the tag.

This allows putting mentions for users and channels in tags as they won't ping anymore.